### DISCUSSION TITLE
Navigating between blocks in the toolbox

### DISCUSSION THREAD

**Discussion by BeksOmega** [RaspberryPiFoundation/blockly-keyboard-experimentation#18] [General]
Related to the [discussion about navigating between top level stacks](https://github.com/google/blockly-keyboard-experimentation/discussions/17), how should navigating between blocks in the toolbox work?

Should there be directional navigation (e.g. vertical toolboxes navigated with up and down keys while horizontal toolboxes are navigated with left and right keys)? Or should the navigation be generic?

**Comment by rachel-fenichel** [comment]
I think that the direction of the keys should match the direction that the cursor moves, which means left and right keys for horizontal toolboxes and up/down for vertical toolboxes.

This also implies using left/right to go through stacks if a workspace is using a horizontal renderer, but that's out of scope for the current work.
